build.sbt

```diff
@@ -31,13 +31,13 @@ lazy val root = (project in file("."))
     name := "sbt-bintray",
     sbtPlugin := true,
     libraryDependencies ++= Seq(
-      "org.foundweekends" %% "bintry" % "0.5.0",
+      "org.foundweekends" %% "bintry" % "0.6.0",
       "org.slf4j" % "slf4j-nop" % "1.7.7"), // https://github.com/softprops/bintray-sbt/issues/26
     resolvers += Resolver.sonatypeRepo("releases")
   )
-  .settings(
-    scalaVersion := "2.12.2",
-    sbtVersion in Global := "1.0.0-M5",
-    scalaCompilerBridgeSource :=
-      ("org.scala-sbt" % "compiler-interface" % "0.13.15" % "component").sources
-  )
+//  .settings(
+//    scalaVersion := "2.12.2",
+//    sbtVersion in Global := "1.0.0-M5",
+//    scalaCompilerBridgeSource :=
+//      ("org.scala-sbt" % "compiler-interface" % "0.13.15" % "component").sources
+//  )
```

Comment:
I think that I need to capture somehow the target `sbt` version here. So what I tried in `plugins.sbt` was something like this (doesn't work):

```
def dispatchScriptedName(sbtVersion: String) = {
  println(sbtVersion)
  CrossVersion.sbtApiVersion(sbtVersion) match {
    case Some((1, _)) => "scripted-plugin_2.12"
    case wat => println(wat); "scripted-plugin"
  }
}

libraryDependencies += "org.scala-sbt" % dispatchScriptedName(sbtVersion.value) % sbtVersion.value
```